src/epd7in5b_v2/mod.rs

```diff
@@ -119,8 +119,9 @@ where
         dc: DC,
         rst: RST,
         delay: &mut DELAY,
+        delay_ms: Option<u8>,
     ) -> Result<Self, SPI::Error> {
-        let interface = DisplayInterface::new(cs, busy, dc, rst);
+        let interface = DisplayInterface::new(cs, busy, dc, rst, delay_ms);
```

Comment:
Since the default delay is the same delay as the one used in here (10ms) nothings needs to be changed

Reply:
The parameter must still be passed